jenkins/job-configs/kubernetes-jenkins-pull/global.yaml

```diff
@@ -1,13 +1,28 @@
+- builder:
+    name: ensure-upload-to-gcs-script
+    builders:
+      - shell: |
+          _script_tmp_path="${WORKSPACE}/_tmp/upload-to-gcs.sh"
+          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
+          mkdir -p "${WORKSPACE}/_tmp"
+          if [[ "${ghprbTargetBranch:-}" == 'master' ]]; then
+            # Force using the checked-out version if we're testing the master branch
+            ln -sf "${WORKSPACE}/kubernetes/${_script_repo_path}" "${_script_tmp_path}"
```

Comment:
Are they checked out to `${WORKSPACE}/kubernetes/hack/...` or just to `${WORKSPACE}/hack/...` on PR Jenkins?